tests/test_main.py

```
@@ -3144,15 +3144,19 @@ async def mock_connect_matrix_fn(*args, **kwargs):
 
         config = {"matrix_rooms": [{"id": "!room:matrix.org"}]}
 
-        async def mock_check_conn():
-            return None
+        check_conn_sentinel = object()
+
+        def mock_check_conn():
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `mock_check_conn`

(ANN202)